runtime/lua/vim/lsp.lua

```diff
@@ -273,12 +273,13 @@ local function text_document_did_open_handler(bufnr, client)
   if not vim.api.nvim_buf_is_loaded(bufnr) then
     return
   end
+  local filetype = nvim_buf_get_option(bufnr, 'filetype')
+  local languageId = client.config.languageId(filetype)
```

Comment:
We should call this `language_id` or `get_language_id`, since that's the naming convention for lua.

Reply:
I'd say as well it should give both the `bufnr` and the `filetype`, since you may want a different language id based on the bufnr itself.

Reply:
Wouldn't `bufnr` be sufficient?
It's easy for the user to get the `filetype` from the `bufnr`.